data/mods/Aftershock/crafting_system.md

```diff
@@ -0,0 +1,67 @@
+# Scrap Crafting System
+
+## Overview
+
+As one of its main  goals, Aftershock allows the crafting of miscellaneous, highly technological devices that have no actual basis in reality.  To make the crafting of these devices more balanced an intuitive, and to make adding more recipes and items easier, aftershock utilizes a crafting a system in which abstracted pieces of increasingly rare scrap are used to craft increasingly useful tools.
+
+
+### Current Scrap Categories
+
+The following table lists the currently planned/implemented scrap categories and their tiers. 
+
+|     Category      |     Tier 1           |    Tier 2                |     Tier 3                 |     Tier 4                |     Tier 5                           |
+| ----------------- | -------------------- | ------------------------ | -------------------------- | ------------------------- | ------------------------------------ |
+| `Circuitry`       | scrap photonics      | photonic circuitry       | photonic computation core  | hypergeometric photonics  | acausal logic permutator             |
+| `Energy Storage`  | Nanowire battery     | ultracapacitor           | Ultracapacitor array       | Superconductive Coil      | zero-point energy extractor          |
+| `Material`        | Composite Superalloy | vacuum cast Carbide      | Nanoprinted Alloys         | crystal forged neutrite   | phase-uneven matter                  |
+| `Cloth`           | E-textile            |                          | graphene weave             | woven metamaterial        |                                      |
+| `Magnet`          | Emag                 | cryo electromagnet       |                            |                           |                                      |
```

Comment:
liquid maglev (a magnetic liquid that repulses and attracts with a specific weight so with no weight on either side it always maintains a specific distance from it's opposite half)

Reply:
Sure good idea